dist/tokens/evo-core.css

```diff
@@ -69,11 +69,19 @@
     --color-lime-5: #86b300;
     --color-lime-6: #4b7d06;
     --color-lime-7: #364f03;
-    --color-ai-green: #4ee04b;
-    --color-ai-blue: #0968f6;
-    --color-ai-purple: #993ee0;
-    --color-ai-red: #ff4242;
-    --color-ai-yellow: #ffd80e;
+    --color-ai-solid-green-strong: #4ee04b;
+    --color-ai-solid-green-subtle: #f1fdf1;
+    --color-ai-solid-green-subtle-dark: #112611;
```

Comment:
Why are `subtle` and `subtle-dark` in `evo-core` instead of `evo-light` and `evo-dark`? I don't believe subtle-dark is ever used in light mode by design system, but maybe I'm missing something.

Reply:
They are base color tokens which is why they are in evo-core. They provide the color that should be used